### ISSUE TITLE
Add image resource name to kp build list output

### ISSUE THREAD

**Issue by mgibson1121** [buildpacks-community/kpack-cli#231]
If you run `kp build list` without providing an image name, you will see a table of all builds for all images, but there is no column for image name. This is confusing for some users.

**Comment by fcaroline2020** [comment]
@mgibson1121 the column "IMAGE RESOURCE" is the image name.

```
BUILD    STATUS      BUILT IMAGE                   REASON     IMAGE RESOURCE
1        SUCCESS     repo.com/image-1:tag          CONFIG     test-image
2        FAILURE     repo.com/image-2:tag          COMMIT+    test-image
3        BUILDING    repo.com/image-3:tag          TRIGGER    test-image
1        BUILDING    repo.com/other-image-1:tag    UNKNOWN    some-other-image
```

We could change the name to IMAGE NAME or would you like another column for BUILD NAME?

**Comment by fcaroline2020** [comment]
Resolved by #218
